### ISSUE TITLE
[Enhancement] Support VoiceOver on buttons with icons

### ISSUE THREAD

**Issue by MortenGregersen** [kirbydesign/designsystem#942]
*NOT Prioritized*
- [ ] For optimal support for VoiceOver this is needed.

*NOT UX Refined*
- [ ] Decide what texts should be used as default.

*NOT Tech Refined*
- [ ] Decide how to specify the configuration. Should the texts eg. be `Observable<string>`, to enable change in language?

**Is your enhancement request related to a problem? Please describe.**
When using VoiceOver (iOS) on a web app using Kirby right now, it can't tell you the purpose of some buttons.

The places I've found up until now are:
- Back button in page
- Close button on modal
- Hide button on drawer

**Describe the solution you'd like**
As mentioned by @jakobe, we should probably configure Kirby with the desired texts for these buttons in Kirby's `forRoot`.

The buttons should have a default text in English.

The texts should be set on the buttons with the `aria-label` attribute.

**Describe alternatives you've considered**
No alternatives considered.

**Additional context**
No additional context.

**Comment by MortenGregersen** [comment]
`KirbyModule` should have a static `setDefaultTranslations` which saves the translations/config statically.

**Comment by MortenGregersen** [comment]
The default texts should be the following:
- Back
- Close
- Hide

**Comment by MortenGregersen** [comment]
There are some texts elsewhere (eg. alert, dropdown) which should also be handled like this.

**Comment by RasmusKjeldgaard** [comment]
https://github.com/kirbydesign/designsystem/issues/905 should be closed by this one.
